//protoc -I=. --plugin=protoc-gen-nanopb=../../generator/protoc-gen-nanopb --nanopb_out=. rad.proto
//syntax = "proto2";

//message DataMessage {
//    repeated uint32 values = 1 [(nanopb).max_size = 40, (nanopb).fixed_length = true];
//    optional uint32 values_count = 2;
//}

// protoc -I=./ --cpp_out=./ rad.proto

syntax = "proto3";
//import "nanopb.proto";

package CAPSTONE;

message MainData {
    uint32 startTime = 1;                            // Time since zeroed out
    float totalCount = 2;                           // TID level (Total Ionizing dose)
    int32 pulseTime = 3;                            //Duration of the read
    repeated uint32 spectrum = 4;                   //spectrum data
}
